Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 
 ARG GO_VERSION=1.21.10
-ARG XX_VERSION=1.4.0
+ARG XX_VERSION=1.5.0
 ARG OSXCROSS_VERSION=11.3-r7-debian
```

Comment:
@crazy-max should this one also be updated to a newer version? https://hub.docker.com/r/crazymax/osxcross/tags

Reply:
Don't think we need to bump to 14.5 yet but `11.3-r8` would be good.